src/frontends/paddle/src/op/sum.cpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs sum(const NodeContext& node) {
+    auto data = node.get_ng_inputs("X");
+    const auto input_type = data[0].get_element_type();
+    const auto shape = data[0].get_shape();
+    auto sum = data[0].get_node_shared_ptr();
+    for (int i = 1; i < data.size(); i++) {
+        // check the consistency of inputs
+        PADDLE_OP_CHECK(node, input_type == data[i].get_element_type(), "The types of inputs are not consistent!");
```

Comment:
Do we have to have these checks? Sounds no need. Paddle won't allow this model, will it? Please have a try.

Reply:
Paddle could detect these error indeed. Removed these CHECK already.